src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -2731,8 +2731,6 @@ components:
           type: string
           description: The identifier of the prison that the prisoner is currently resident at.
           example: BXI
-        type:
-          $ref: '#/components/schemas/ConversationType'
```

Comment:
I think I included this by accident. There's no user need to update the conversation type (at least not currently).